pkg/config/expand.go

```diff
@@ -868,7 +868,11 @@ func expandSimpleVariable(context varContext, trackModuleGraph bool) (string, er
 		// ensure that the target module outputs the value in the root module
 		// state and not just internally within its deployment group
 		if !slices.ContainsFunc(toMod.Outputs, func(o modulereader.OutputInfo) bool { return o.Name == varRef.name }) {
-			toMod.Outputs = append(toMod.Outputs, modulereader.OutputInfo{Name: varRef.name})
+			toMod.Outputs = append(toMod.Outputs, modulereader.OutputInfo{
+				Name:        varRef.name,
+				Description: fmt.Sprintf("Automatically-generated output from module '%s'", toMod.ID),
```

Comment:
This is just slightly different that the [message that is written](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/8bac6ecaeb2d50dcc470321194e5bbe6a8c9f7c1/pkg/modulewriter/tfwriter.go#L107) for user generated outputs. I am wondering if we shouldn't explain a bit more that this was generated by IGC requirements. Thoughts?

Reply:
I would like us to discuss what lexicon/terminology we will present to users and align our messaging on that. I could imagine "for later deployment groups" instead of "from...". Would you prefer that?

This value doesn't show up anywhere except in outputs.tf so it's of limited utility.

Reply:
Offline discussion resulted in:

```
"Automatically-generated output exported for use by later deployment groups"
```